content/admin/manage_orgs/Licensing.md

```diff
@@ -4,51 +4,53 @@ description: "Overview of Licensing visibility"
+To apply a Protect license to an an individual server, click the **UNLICENSED** beside the server name, or click the arrow in the grid row and choose **Apply Protect License** from the dropdown menu. To apply Protect licenses to multiple servers, select the checkboxes in the appropriate grid rows, click the badge icon in the batch actions menu above the grid, and choose **Apply Protect License** from the menu. Click the **Apply License** button in the confirmation dialog. 
 
-<a href="assets/images/Licensing_OrgSettings.png" rel="lightbox" title="Organization Settings License Summary"><img class="thumbnail" src="assets/images/Licensing_OrgSettings.png"/></a>
+Once a license is applied, the Protect toggle is enabled in the grid row(s).
 
+<a href="assets/images/Servers-unlicensed.png" rel="lightbox" title="Apply a Protect license"><img class="thumbnail" src="assets/images/Servers-unlicensed.png"/></a>
 
-## Applying And Removing Protection Licenses
+>**Note:** Protect licenses are currently available for Java servers only.
 
-On the **Servers** page, a user must choose which server(s) they wish to apply a Protection license to.  Unlicensed servers will have a link to apply protection next to their name, and the toggle in the **Protect** column will be switched off and disabled.  The **Protect** link allows for quick and easy licensing of an individual server.  Applying a license can also be done via the row menu dropdown or in a batch by selecting servers and clicking the **Protection Licenses** (shield) in the batch action buttons.  One or more licenses may also be removed using this dropdown. Once a license is applied, the toggle will be enabled and Protection may be turned on.
+#### Available licenses
 
-<a href="assets/images/Licensing_Servers.png" rel="lightbox" title="Applying / Removing a Protection License"><img class="thumbnail" src="assets/images/Licensing_Servers.png"/></a>
+If your organization has consumed all allocated licenses, or if the organization doesn't have enough licenses available for the number of servers selected, the option to apply a license is disabled. Hover over the disabled selection to **Apply Protect Licenses** for more information. 
 
->**Note:** Protection licenses are currently available for Java servers only.
+> **Note:** To return a license to the pool of available licenses for the organization, you must remove it from a server. 
 
+### Remove licenses 
 
-### Lack Of Available Licenses
+To remove licenses from an individual server, click the arrow in the grid row and choose **Remove Protect License** from the dropdown menu. To remove Protect licenses to multiple servers, select the checkboxes in the appropriate grid rows, click the badge icon in the batch actions menu above the grid, and choose **Remove Protect License** from the menu. Click the **Remove** button in the confirmation dialog. 
```

Comment:
I don't like saying "the arrow in the grid row". It's actually a caret, but regardless, let's refer to it as what it really is: a row action dropdown menu (as opposed to the batch action menu). If you're saying "arrow" to be more specific in regards to location, we can probably word it such that its location is still conveyed. Ex: "Activate the row's dropdown menu on the right" or "Open the row action dropdown menu on the right side of the grid." etc.

Reply:
"To remove Protect licenses **from** multiple servers..."

Reply:
We've always referred to this as the "batch action menu" (not actions). Not a huge deal but sounds better and is easier to say IMO. :)